app/admin/page.tsx

```
@@ -22,23 +23,72 @@ export default function AdminPage() {
+                <FileText className="h-4 w-4 text-green-400" />
+                <span className="text-sm text-white">Content</span>
+              </Link>
+            </div>
+          </div>
+
+          {/* Account actions */}
+          <div className="space-y-3">
+            <button
+              onClick={() => router.push('/')}
+              className="w-full rounded-full border border-white/20 bg-white/5 px-6 py-3 text-white transition-colors hover:bg-white/10"
+            >
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)